tests/test_log_likelihood.py

```diff
@@ -0,0 +1,88 @@
+"""Assert values log-likelihood function under ambiguity (speed-up)"""
+import os
+from pathlib import Path
+
+import numpy as np
+import pytest
+import respy as rp
+from respy.likelihood import get_crit_func
+from respy.simulate import get_simulate_func
+
+subdir = Path(f"{os.environ['PROJECT_ROOT']}/tests/resources")
+
+
+# Test: Assert likelihood value for different model specifications (options, params)
+# Includes ambiguity for iteration i > len(model_spec_params)/2
+# Reference data was generated with inclusion of ambiguity but w/o speedup
+def test_loglike_around_models():
+    # Load model specifications and associated loglike values
+    model_spec_params = np.load(
```

Comment:
Please don't use `*.npy` nd rely on classical `pandas` dataframes with mutlindexing instead. Just add the counter for the random model specification as its first level. Also, you can then add the likelihood infromation directly into that file as well so we only have the usual `params` and `model_spec` dataframes. This is a little advanced `pandas`, so please reach out to @MaxBlesch  if you need help or clarification and then afterwards call me on my cell if needed.

.. honestly, I had never heard about `*npy` before.

Reply:
Thanks - think I've rewritten it accordingly.
Will send you a short follow-up on zu.